internal/certdb/certdb.go

```diff
@@ -53,8 +53,8 @@ type CertificateRequest struct {
 type User struct {
 	ID          int    `json:"id"`
 	Username    string `json:"username"`
-	Password    string `json:"password"`
-	Permissions int    `json:"permissions"`
+	Password    string `json:"-"`
+	Permissions *int   `json:"permissions"`
```

Comment:
a `-` will always remove the field. `omitempty` will only remove the field when the value is a nil value. So if we set the password to `""` above, we can remove it from the json string we marshalled, but still keep the ability to look at the password (we will need to check it when logging in)
```suggestion
	Password    string `json:"password,omitempty"`
	Permissions *int   `json:"permissions"`
```

Reply:
✅